src/main/kotlin/uk/gov/justice/digital/hmpps/digitalprisonreportingmi/service/ExternalMovementService.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.digital.hmpps.digitalprisonreportingmi.service
+
+import jakarta.validation.ValidationException
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.digitalprisonreportingmi.data.ExternalMovementRepository
+import uk.gov.justice.digital.hmpps.digitalprisonreportingmi.model.ExternalMovement
+
+@Service
+data class ExternalMovementService(val externalMovementRepository: ExternalMovementRepository) {
```

Comment:
Would it be worth moving the paging and sorting into the repository? I'm thinking that when we come to add the real RedShift data, they'll be happening in the DB. That way we can just change which data repo the service points to, which will have a similar interface.

Reply:
True, I can change that. I thought of this initially but then added it to the service as the pagination would likely be done through a SQL query on the repository side later and the service would just call the repository while the service pagination code would be removed. 
Happy to move this to the repository though as indeed this is where this functionality will take place.